addons/project_mrp/models/project_project.py

```diff
@@ -87,13 +101,16 @@ def _get_profitability_items(self, with_action=True):
     def _get_stat_buttons(self):
         buttons = super(Project, self)._get_stat_buttons()
         if self.user_has_groups('mrp.group_mrp_user'):
-            buttons.extend([{
-                'icon': 'flask',
-                'text': _lt('Bills of Materials'),
-                'number': self.bom_count,
-                'action_type': 'object',
-                'action': 'action_view_mrp_bom',
-                'show': self.bom_count > 0,
-                'sequence': 35,
-            }])
+            self.sudo()._append_bom_button(buttons)
```

Comment:
The sudo is used because of a company issue with the compute. When the get_stat_button is called and use a compute field, it needs a sudo() to bypass company access right (e.a. if a ticket is in company A, and the user does not have access to that company, the get_stat_button will fails because of access rights.). Though, in the case of the project_forecast module, the sudo() is indeed not needed because the computed field already has the compute_sudo attribute set to True. The update has been done accordingly.